### PR TITLE
fix(issuer): createAndSendAnalyticsEvent

### PR DESCRIPTION
**Pull request by silvicir**
#### List of Changes

Swapped parameters in `createAndSendAnalyticsEvent` function in workspace `io-func-sign-issuer`

### CODE REVIEW

**Review by grausof — APPROVED** [review]
🔝

File: apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export const CreateSignatureRequestHandler = H.of((req: H.HttpRequest) =>
     ),
     RTE.chainW(insertSignatureRequest),
     RTE.chainFirstW((request) =>
-      pipe(EventName.SIGNATURE_CREATED, createAndSendAnalyticsEvent(request))
+      pipe(request, createAndSendAnalyticsEvent(EventName.SIGNATURE_CREATED))
```
Comment (gunzip) [inline apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:73]: you can just use

`RTE.chainFirstW(createAndSendAnalyticsEvent(EventName.SIGNATURE_CREATED))`

Comment (silvicir) [reply apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:73]: Actually when I tried it did not work. `RTE.chainFirstW(createAndSendAnalyticsEvent(EventName.SIGNATURE_CREATED))` changes the data type

Comment (gunzip) [reply apps/io-func-sign-issuer/src/infra/http/handlers/create-signature-request.ts:73]: weird!

**Pull request merged**